namui-prebuilt/src/button.rs

```diff
@@ -1,10 +1,10 @@
 use crate::{simple_rect, typography::center_text_full_height};
 use namui::prelude::*;
 
-fn attach_text_button_event<'a>(
+fn attach_text_button_event(
     ctx: &mut ComposeCtx,
     mouse_buttons: Vec<MouseButton>,
-    on_mouse_up_in: Box<dyn 'a + Fn(MouseEvent)>,
+    on_mouse_up_in: impl Fn(MouseEvent<'_>),
```

Comment:
I just followed `ComposeCtx`'s `attach_event` parameter type. Because it seems more flexible than `&dyn fn`

Reply:
You should change this callbacks to FnOnce at least. It's Fn so may make a copy problem.
Then, you may need lifetime because they need to know how 'Once' it is.

Reply:
Fixed with 745b9ee